extensions/sql-migration/config.json

```diff
@@ -1,12 +1,12 @@
 {
 	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/{#version#}/microsoft.sqltools.migration-{#fileName#}",
 	"useDefaultLinuxRuntime": true,
-	"version": "4.5.0.12",
+	"version": "4.5.0.28",
```

Comment:
should we proactively bump the version of the extension to release these changes?

Reply:
That's a good idea. As I mentioned offline, these changes don't need to be released urgently -- it's more so for devs who are locally making changes and need to update the JSON-RPC migration service. I'll be sending out a PR for login migrations telemetry soon and will bump up the extension version number.